packages/liveblocks-yjs/src/index.ts

```diff
@@ -24,6 +25,13 @@ const DefaultOptions: ProviderOptions = {
   autoloadSubdocs: false,
 };
 
+enum SyncStatus {
+  Loading = "loading",
+  NotLoaded = "not-loaded",
```

Comment:
Minor nit, but I prefer to flip these to make them match the order of state transitions over time:

```suggestion
  NotLoaded = "not-loaded",
  Loading = "loading",
```

Note that everywhere else we use `not-loaded` as the initial state, think NotYetEvenStartedToLoad.

Reply:
okay, I removed not-loaded from yjs provider because it will never be in that state if the provider exists, then the room exists and it will be loading. The hooks that implement this, i.e. getEditorStatus do implement not-loaded.